index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <title>Practice Website</title>
+  </head>
+
+  <body>
+    <div class="top">
+      <div class="header">
+        <div class="header-logo">Header Logo</div>
+  
+        <div class="links">
+          <ul>
+            <li><a href="google.com">header link one</a></li>
+            <li><a href="google.com">header link two</a></li>
+            <li><a href="google.com">header link three</a></li>
+          </ul>
+        </div>
+      </div>
+  
+      <div class="spotlight">
```

Comment:
A [section] semantic element would be good here. They are good for most of the elements that go between the header and footer